src/nft/ZoraCreator1155Impl.sol

```diff
@@ -372,15 +372,35 @@ contract ZoraCreator1155Impl is
     /// @param tokenId The token ID to mint
     /// @param quantity The quantity of tokens to mint
     /// @param minterArguments The arguments to pass to the minter
-    function mint(IMinter1155 minter, uint256 tokenId, uint256 quantity, bytes calldata minterArguments) external payable nonReentrant {
+    /// @param finder The user or platform that finds the sale
+    /// @param lister The originator of the sale
+    function mint(
+        IMinter1155 minter,
+        uint256 tokenId,
+        uint256 quantity,
+        bytes calldata minterArguments,
+        address finder,
```

Comment:
I'd be hesitant to force all users of the contract to now pass these arguments additionally if they're only used on a small % of contracts. Adds to gas costs + complexity of the api.  I'd lean to adding another method that allows for these additional arguments.